3D/senior-csharp-test/README.md

```diff
@@ -6,14 +6,14 @@ Unity3D 2018.3.12 ( https://download.unity3d.com/download_unity/8afd630d1f5b/Win
 
 ## Guidelines
 
-We would like to test your ability to understand existing code, as well as your capacity of either implementing unit tests or learning how to do it. 
-Our tests are implemented using Zenject and NUnit, you have all the necessary includes and assembly definitions provided. 
+We would like to test your ability to understand existing code, as well as your capacity of either implementing unit tests or learning how to do it. \
+Our tests are implemented using Moq and NUnit, you have all the necessary includes and assembly definitions provided. 
 
 ## Content
 
-We have provided a small module that implements saving files to a cache, and retrieving them. 
-The main class is FileCacheStorage, and this is the one we want to test. 
-We have provided the FileCacheStorageTest file to get you started, so all you need to do is write your test functions inside. 
-We aim at unit testing rather than integration or end-to-end tests.
-The naming convention we usually follow for unit tests is `MyFunction_DoThis_When_SomethingHappens`, allowing to clarify the intent of the test.
+We have provided a small module that implements saving files to a cache, and retrieving them. \
```

Comment:
hm, no saving any more. Maybe it's the right place to mention that the class is simplified and the cache save is not implemented?

Reply:
Indeed, I'll try to explain that it's the cache file save that is not implemented (because Add() still save cache files on the FS)

Reply:
done

Reply:
true

Reply:
I just wrote that the cache save is not implemented because finally I think it's still understandable